server/services/CommonService.ts

```diff
@@ -14,6 +14,8 @@ import {
 } from "../../../../src/core/server";
 import { IAPICaller } from "../../models/interfaces";
 
+const VALID_METHODS = ["HEAD", "GET", "POST", "PUT", "DELETE"];
+
```

Comment:
should we support all standard http methods mentioned here https://www.rfc-editor.org/rfc/rfc7231#section-4.1. ?

CONNECT   
DELETE   
GET  
HEAD  
OPTIONS 
POST  
PUT 
TRACE

Reply:
sounds good.